AdornUI-Stylesheets/navigation.css

```diff
@@ -0,0 +1,85 @@
+.navigation-container {
+  width: auto;
+  justify-content: space-around;
+  align-items: center;
+  background-color: var(--secondary-bg-color);
+}
+
+.navigation-item-2 {
+  width: 30rem;
+  height: 2rem;
+  border-radius: 10px;
+}
+
+.navigation-item-3 {
+  justify-content: space-around;
+  align-items: center;
+  margin: 1rem;
+  gap: 2rem;
+}
+
+.fa-bars {
+  display: none;
+}
+
+.search-bar {
+  font-size: 2rem;
+  background-color: var(--grey);
+  border-radius: 10px;
+  padding: 0.5rem;
+}
+
+.search-input {
+  font-size: 2rem;
+  outline: none;
+  background-color: var(--grey);
+  border: none;
+  border-radius: 10px;
+}
+
+::placeholder {
+  color: var(--primary-text-color);
+}
+
+.nav-search-icon {
+  position: absolute;
+  left: 2px;
+  top: 5px;
+  font-size: 1.5rem;
+  color: gray;
```

Comment:
use hexcode for colors i guess to maintain consistency as you might have used hexcode at other palces. Someone suggested me to do so. If you think it's fine then ignore this